Project.toml

```diff
@@ -48,7 +48,7 @@ StatsBase = "0.31, 0.32, 0.33, 0.34"
 StatsFuns = "0.8, 0.9, 1"
 LinearAlgebra = "1.6"
 Random = "1.6"
-julia = "1.6"
+julia = "1.6.7"
```

Comment:
Tests seem to pass with 1.6.7 but didn't with 1.6.0. Not sure which dependency is the problem.